proto/backup.proto

```diff
@@ -124,6 +125,26 @@ message S3 {
     string secret_access_key = 9;
 }
 
+// GCS storage backend saves files into google cloud storage.
+message GCS {
+    string endpoint = 1;
+    string bucket = 2;
+    string prefix = 3;
+    string storage_class = 4;
+    // acl json: like `[
```

Comment:
I think this is a serialized json string. It doesn't have to be the format below. So I suggest modifying the comment. And also it's better to add a documentation link to these two fields since many people not knowing this.

Reply:
Updated.